### DISCUSSION TITLE
Welcome to ko Discussions!

### DISCUSSION THREAD

**Discussion by stevewgr** [ko4life-net/ko#95] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Deed8855** [comment]
What order do we start the servers in i know AI server first but what are rest? Thanks!

**Reply by xGuTeK** [reply]
AI Server, Ebenezer, Ajuard, Login Server
